recipe/conda_build_config.yaml

```diff
@@ -25,24 +29,28 @@ runtime_version:
 # reported in the VS help->about UI.  It is perhaps a more readily
 # referenceable number.
 update_version:
+ - 4
  - 2
  - 11
  - 8
  - 9
 # This is the version number reported by cl.exe
 cl_version:
+ - 14.34.31933
```

Comment:
```suggestion
 - 19.34.31933
```

Reply:
wow, thanks & sorry for that stupid mistake.